<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextDialog</class>
 <widget class="QDialog" name="TextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_txt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>字体:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_width">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>宋体</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>微软雅黑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>仿宋</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>楷体</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新宋体</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑体</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Times New Roman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Verdana</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vrinda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tunga</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vani</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字框宽度：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Width">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
